modules/apps/staging/staging-bar-web/src/main/java/com/liferay/staging/bar/web/internal/util/StagingBarControlMenuJSPDynamicIncludeUtil.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.staging.bar.web.internal.util;
+
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.Layout;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.Constants;
+import com.liferay.portal.kernel.util.ParamUtil;
+import com.liferay.portal.kernel.util.WebKeys;
+import com.liferay.staging.bar.web.internal.servlet.taglib.ui.StagingBarControlMenuJSPDynamicInclude;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * @author Valmir Junior
+ */
+public class StagingBarControlMenuJSPDynamicIncludeUtil {
```

Comment:
@Valmir-unicap Could you rename it to `StagingBarControlMenuUtil` since it's not just for the dynamic include.